### PR TITLE
refactor - use pandas apply and map function instead of for loop

### PR DESCRIPTION
**Pull request by iloveonsen**
## Overview
- baseline 코드 리뷰 완료하였습니다.
- Dataloader 에서 for 문 사용했던 부분을 dataframe 에서 직접 변경하도록 변경하였습니다.

## Change Log
- `dataloader.py`
  - `load_data`  에서 apply 함수를 사용하였습니다.
  - `label_to_num` 에서 map 함수를 사용하였습니다.

## To Reviewer
- train 과 test set 둘다 확인했을때 데이터상에서 어떠한 오류도 없었음을 확인하였습니다.

## Issue Tags
- Closed: #
- See also: #

### CODE REVIEW

**Comment by DonghaeSuh** [comment]
추가하겠습니다~
